### PR TITLE
For Ember Times No. 119: Writeup on @Pzuraq 's Blog Post on Mut and 2 Way Data Bindings

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-10-11-the-ember-times-issue-119.md [hunk 1]
```diff
@@ -54,13 +55,43 @@ Read the [full blog post](https://embersteps.com/promises-and-ui-states-in-ember
...
-<consider adding some bold to your paragraph>
+A handy **helper** has existed in the framework's API since the early days of Ember 1.13. It allows to specify that a certain value - which was passed down by a parent component or controller - **may be mutated** by a child component. This is [the so-called `{{mut}}` helper](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/mut?anchor=mut).
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Most Ember developers are familiar with the usage of `{{mut}}` from scenarios as the following one:
+
+```handlebars
+<!-- app/templates/components/my-component.hbs -->
+<input value={{@passedDownValue}} oninput={{action (mut @passedDownValue) value="target.value"}} />
```
Comment (ijlee2) [inline source/2019-10-11-the-ember-times-issue-119.md]: For small-width resolution, can we break this line into multiple like this?
```suggestion
<input
  value={{@passedDownValue}}
  oninput={{action (mut @passedDownValue) value="target.value"}}
/>
```

File: source/2019-10-11-the-ember-times-issue-119.md [hunk 2]
```diff
@@ -54,13 +55,43 @@ Read the [full blog post](https://embersteps.com/promises-and-ui-states-in-ember
...
+    updateValue(newValue) {
+      this.set('passedDownValue', newValue);
+    },
+  },
+});
+```
+
+```handlebars
+<!-- app/templates/components/my-component.hbs -->
+<input value={{@passedDownValue}} oninput={{action "updateValue" value="target.value"}} />
```
Comment (ijlee2) [inline source/2019-10-11-the-ember-times-issue-119.md]: If yes to above, then, similarly,
```suggestion
<input
  value={{@passedDownValue}}
  oninput={{action "updateValue" value="target.value"}}
/>
```

**Review by pzuraq — APPROVED** [review]

File: source/2019-10-11-the-ember-times-issue-119.md [hunk 3]
```diff
@@ -54,13 +55,43 @@ Read the [full blog post](https://embersteps.com/promises-and-ui-states-in-ember
...
+```
+
+With [Ember Octane coming soon](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) and the new Glimmer Component API [enforcing unidirectional data flow](https://blog.emberjs.com/2019/03/14/coming-soon-in-ember-octane-part-5.html), many developers might wonder how the `{{mut}}` helper still fits into the overall picture. Are there any issues with using `{{mut}}` in future Octane apps? And are there still plans to [deprecate the helper](https://github.com/emberjs/rfcs/issues/538) or 2-way-binding APIs from the framework in general?
+
+You will find answers to all of these questions in an [excellent deep-dive into `mut` and 2-way-bindings](https://www.pzuraq.com/on-mut-and-2-way-binding/) by [Chris Garrett (@pzuraq)](https://github.com/pzuraq). This blog post will clarify which data-binding framework APIs will persevere in the foreseeable future (including Ember Octane) and which questions are still open in regards to 2-way binding components commonly used in Ember apps today.
```
Comment (ijlee2) [inline source/2019-10-11-the-ember-times-issue-119.md]: I don't think persevere is the right word here (to me, persevere means endure through difficulties). Maybe remain instead?
```suggestion
You will find answers to all of these questions in an [excellent deep-dive into `mut` and 2-way-bindings](https://www.pzuraq.com/on-mut-and-2-way-binding/) by [Chris Garrett (@pzuraq)](https://github.com/pzuraq). This blog post will clarify which data-binding framework APIs will remain for the foreseeable future (including Ember Octane) and which questions are still open in regards to 2-way binding components commonly used in Ember apps today.
```

**Review by ijlee2 — APPROVED** [review]
Great summary and interpretation of Chris' blog post. I had read it myself (left with some confusion) and thought your code examples helped!

**Pull request merged**